python/doc/examples/functional_modeling/vectorial_functions/plot_createMultivariateFunction.py

```diff
@@ -0,0 +1,168 @@
+"""
+=============================
+Create multivariate functions
+=============================
+We can create multivariate functions by different methods. One of the methods is to gather
+multivariate functions. The other one is to create a function having multivariate input using
+tensorization of univariate functions. In this example, we present both methods.
+"""
+# %%
+#
+# Import the required modules
+import openturns as ot
+
+# %%
+# Description
+# ===========
+#
+# We want to build some multivariate functions :math:`f : \mathbb{R}^d \mapsto \mathbb{R}^q`.
+#
+# We can do that:
+#
+# - Case 1: using a finite collection of multivariate functions :math:`f_i:  \mathbb{R}^d \mapsto \mathbb{R}^q`,
+# - Case 2: using a tensorization of a univariate functions family,
+# - Case 3: using a tensorization of an orthogonal univariate polynomials family,
+# - Case 4: using a tensorization of an orthogonal univariate functions family.
+#
+#
+# Case 1: we use a finite collection of multivariate functions
+# ============================================================
+#
+# In that case, we have a  finite collection of multivariate functions  :math:`f_i : \mathbb{R}^d \mapsto \mathbb{R}^q`.
+# We use the :class:`~openturns.Basis` class.
+#
+#
+# For example, we consider :
+#
+# .. math::
+#     \begin{array}{lcl}
+#       f_1(x_1, x_2) & = & (x_1^2, x_2^2)\\
+#       f_2: (x_1, x_2) & = & (x_1+1, 2x_2)\\
+#       f_3: (x_1, x_2) & = & (cos(x_1x_2), x_2^3)
+#     \end{array}
+
```

Comment:
```suggestion

import openturns as ot

```

Reply:
idem

Reply:
I agree with Joseph. This comment is unnecessary.